core/modules/util/threadSafe.h

```diff
@@ -74,17 +74,20 @@ class Flag {
     /** Sets flag value to 'val' and returns the old value of flag.
      */
     virtual T set(T val) {
```

Comment:
Agree -- let's please drop a TODO: comment and put an issue on the backlog so we don't forget?